### PR TITLE
Add SCSS, theming functionality, and restructure of files

### PR DESCRIPTION
**Pull request by justin-dupre**
Rough draft for file structure, especiallly with the .scss. Want to try to use this for a bit and see how it works.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "axios": "^0.19.2",
+    "node-sass": "^4.14.1",
```
Comment (justin-dupre) [inline package.json:10]: required to use scss

File: src/App.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ export default function App() {
   const dispatch = useDispatch();
 
   useEffect(() => {
+    document.documentElement.classList.add(`theme-light`);
```
Comment (justin-dupre) [inline src/App.js:23]: This initalized app with light mode enabled, needed while we don't store user preferences (ie settings endpoint)

File: src/components/auth/LoginForm.js [hunk 3]
```diff
@@ -21,36 +19,29 @@ export default function Login() {
...
 
   return (
-    <div className='page'>
-      <h2>Welcome Back</h2>
-      {error && (
-        <ErrorNotice message={error} clearError={() => setError(undefined)} />
-      )}
-      <form className='form' onSubmit={submit}>
+    <div>
+      <form className='form login-form' onSubmit={submit}>
```
Comment (justin-dupre) [inline src/components/auth/LoginForm.js:31]: Made this component just for the form, not the whole page

File: src/components/auth/PrivateRoute.js [hunk 4]
```diff
@@ -12,9 +12,9 @@ export default function PrivateRoute({ exact, path, component }) {
     }
   }, [history, user]);
 
-  return (
+  return !!user ? (
```
Comment (justin-dupre) [inline src/components/auth/PrivateRoute.js:15]: this check will make this route not show anything in the brief time you try to accees a private route when not logged in. Purely UI enhancement

File: src/components/auth/RegisterForm.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
```
Comment (justin-dupre) [inline src/components/auth/RegisterForm.js:1]: This file was just renamed from Register.js to RegisterForm.js

File: src/components/auth/_auth.scss [hunk 6]
```diff
@@ -0,0 +1 @@
+@import './loginForm';
```
Comment (justin-dupre) [inline src/components/auth/_auth.scss:1]: This willl be where all .scss files are imported from this auth component directory

File: src/components/common/header/Header.js [hunk 7]
```diff
@@ -1,10 +1,12 @@
...
-import AuthOptions from '../auth/AuthOptions';
-import pear from './pear.png';
+import AuthOptions from '../../auth/AuthOptions';
+import pear from '../../../pear.png';
+import { useSelector } from 'react-redux';
 
 export default function Header() {
-  return (
+  const user = useSelector((state) => state.userReducer.user);
+  return !!user ? (
```
Comment (justin-dupre) [inline src/components/common/header/Header.js:9]: Made header only show when logged in, purely UI decision to make pre-login more flashy

File: src/scss/_base.scss [hunk 8]
```diff
@@ -0,0 +1,87 @@
+@import './colors';
```
Comment (justin-dupre) [inline src/scss/_base.scss:1]: This base scss file is boiler plate and will need to be enhanced / looked at

**Review by Ultoman — APPROVED** [review]
looks good

**Pull request merged**